rust/ql/consistency-queries/PathResolutionConsistency.ql

```diff
@@ -10,22 +10,17 @@ private import codeql.rust.internal.PathResolution
 private import codeql.rust.internal.PathResolutionConsistency as PathResolutionConsistency
 private import codeql.rust.elements.Locatable
 private import codeql.Locations
-import PathResolutionConsistency
 
 class SourceLocatable extends Locatable {
-  Location getLocation() {
-    if super.getLocation().fromSource()
-    then result = super.getLocation()
-    else result instanceof EmptyLocation
-  }
+  SourceLocatable() { this.fromSource() }
```

Comment:
So if I'm understanding this correctly, for locatables outside of the source we're now generally not doing path resolution consistency checks, whereas previously we were assigning them the empty location and hoping for the best?